### PR TITLE
Document how to build release engine for Android

### PR DESCRIPTION
**Pull request by Hixie**
I couldn't find any docs on how to do this.

### CODE REVIEW

**Comment by Hixie** [comment]
@jason-simmons I haven't added anything about --deploy yet, but this is a start.

**Comment by jason-simmons** [comment]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -87,11 +87,29 @@ connected Android device, from that directory you would run:
...
 If you're going to be debugging crashes in the engine, make sure you add
 `android:debuggable="true"` to the `<application>` element in the
 `android/AndroidManifest.xml` file for the Flutter app you are using
 to test the engine.
 
+To build a release engine, run the following steps, again from the
+`src` directory created in the steps above:
+
+ * `gclient sync` to update your dependencies.
+ * `./sky/tools/gn --android --release` to prepare your build files to build an Android Release binary.
```
Comment (abarth) [inline CONTRIBUTING.md]: I think @jason-simmons removed the `--release` flag from this script.  Release is now just the absence of `--debug`.  Maybe we should add `--release` as an alias for user friendliness?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -87,11 +87,29 @@ connected Android device, from that directory you would run:
...
+ * `ninja -C out/android_Release` to actually build an Android Release binary.
```
Comment (abarth) [inline CONTRIBUTING.md]: Also, there's no `_Release` in the directory name anymore.  It's just `out/android`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -62,8 +62,8 @@ Building and running the code
 Run the following steps, from the `src` directory created in the steps above:
 
  * `gclient sync` to update your dependencies.
- * `./sky/tools/gn --android` to prepare your build files.
- * `ninja -C out/android_Debug` to build an Android Debug binary.
+ * `./sky/tools/gn --android` to prepare your build files to build an Android Debug binary.
+ * `ninja -C out/android_Debug` to actually build an Android Debug binary.
```
Comment (abarth) [inline CONTRIBUTING.md]: I believe `Debug` is now lower case: `debug`

File: CONTRIBUTING.md (hunk 2 above)
Comment (jason-simmons) [inline CONTRIBUTING.md]: The "develop" runtime mode is selected by default.

"gn --android" will output to out/android_develop

Use "gn --android --runtime_mode=release" to build a release mode engine (which will live in out/android_release)

File: CONTRIBUTING.md (hunk 3 above)
Comment (jason-simmons) [inline CONTRIBUTING.md]: Use the GN --debug flag to build a debug engine

"gn --android --debug" outputs to out/android_develop_debug

**Comment by Hixie** [comment]
Obsoleted by https://github.com/flutter/engine/pull/2676

**Pull request closed without merging**